tsconfig.json

```diff
@@ -1,15 +1,10 @@
 {
   "compilerOptions": {
-    "target": "es6",
```

Comment:
You should add `"target": "ES6" ` and `"module": "CommonJS"` to `tsconfig.json`.